Makefile

```diff
@@ -1,20 +1,15 @@
-DEPLOY_BUCKET=18f-dap
 
 production:
 	bundle exec jekyll build
 
+staging:
+	bundle exec jekyll build --config=_config.yml,_staging.yml
+
 dev:
 	bundle exec jekyll serve --watch --config=_config.yml,_development.yml
 
-deploy:
-	make production && \
-	s3cmd put --recursive -P -M \
-	--add-header="Cache-Control:max-age=0" _site/* \
-	s3://$(DEPLOY_BUCKET)/ && \
-	s3cmd put -P --mime-type="text/css" \
-	--add-header="Cache-Control:max-age=0" _site/css/*.css \
-	s3://$(DEPLOY_BUCKET)/css/ && \
-	s3cmd put -P --mime-type="text/css" \
-	--add-header="Cache-Control:max-age=0" _site/css/vendor/css/*.css \
-	s3://$(DEPLOY_BUCKET)/css/vendor/css/
+deploy_production:
+	make production && cf push analytics
 
+deploy_staging:
+	make staging && cf push analytics-staging
```

Comment:
To deploy, use `make deploy_staging` or `make deploy_production`. 

To just build the staging or production sites without deploying, use `make staging` or `make deploy`.